contribution/lang/de.json

```diff
@@ -3268,7 +3268,7 @@
  	 	 	 	"eng": "calibration"
  	 	 	},
  	 	 	"enemyGangsAndMark": {
- 	 	 	 	"trans": "Feinde Gangs und Ausrüstungsmerkmale",
+ 	 	 	 	"trans": "Gangs und Ausrüstungsmerkmale",
```

Comment:
This should read as Enemy gangs ( as its talking about the npc gang's Voodoo etc )